hack/print-workspace-status.sh

```diff
@@ -20,7 +20,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_ROOT
```

Comment:
I'm not sure we need to export this at all.

Reply:
Removed it.